bootstrap/dev/venv.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh -xe
 # Developer virtualenv setup for Let's Encrypt client
 
+export VENV_ARGS="--python python2"
```

Comment:
I'd do at this stage `PYTHON=${PYTHON:-python2}` in `_venv_common.sh` so that people can at least override it. But you don't have to do it in this PR.